docs: Refactor create-notebook page for GPU link (#132)
canonical/data-science-stack: docs/how-to/jupyter-notebook/create-notebook.rst

- DSS CLI installed on your workstation
- DSS initialized

Select an Image
---------------

> Before creating a notebook, you need to select an image that includes the packages and toolkits you need.

Reviewer: ```suggestion
To create a notebook, you need to select an image that includes the packages and toolkits you need.
```

> To see a list of recommended images and their aliases, see:

Reviewer: ```suggestion
To see a list of recommended images and their aliases, run:
```

.. code-block:: bash

    dss create --help

> The help text includes a list of recommended images and aliases so you don't need to type the full image name.

Reviewer: ```suggestion

```

> For this guide, we will use the image `kubeflownotebookswg/jupyter-scipy:v1.8.0`

Reviewer: ```suggestion
For this guide, the image `kubeflownotebookswg/jupyter-scipy:v1.8.0` is used.
```

Launch the Notebook
-------------------

> Create a new notebook using ``dss create``:

Reviewer: ```suggestion
You can create and launch a new notebook using ``dss create``:
```

.. code-block:: bash

    dss create my-notebook --image kubeflownotebookswg/jupyter-scipy:v1.8.0

> This will pull the notebook image and start a Notebook server, printing the URL of the notebook once complete.  Expected output:

Reviewer: ```suggestion
This command pulls the notebook image and start a Notebook server, printing the URL of the notebook once complete.  You should expect this output:
```

.. code-block:: none

    [INFO] Executing create command
    [INFO] Waiting for deployment test-notebook in namespace dss to be ready...
    [INFO] Deployment test-notebook in namespace dss is ready
    [INFO] Success: Notebook test-notebook created successfully.
    [INFO] Access the notebook at http://10.152.183.42:80.

Access the notebook
-------------------

> To :doc:`access the Notebook </how-to/jupyter-notebook/access-ui>`, use the URL provided in the output.

Reviewer: ```suggestion
To :doc:`access the notebook </how-to/jupyter-notebook/access-ui>`, use the URL provided in the launching phase.
```

> Conclusion

Reviewer: I am not sure these conclusions sections are actually needed for how-to guides. I will add this to the backlog list.
